docs/adguard-for-android/solving-problems/low-level-settings.md

```diff
@@ -214,4 +218,4 @@ Here you can enter the IP address that will be used to create a TUN interface. B
 
 #### Detect Samsung Pay
 
-Korean users may encounter issues with Samsung Pay when AdGuard is enabled. Activate this feature to pause AdGuard's protection and use the Samsung Pay app seamlessly.
+Korean users may encounter issues with Samsung Pay when AdGuard is enabled. This option pauses AdGuard's protection while Samsung Pay is used.
```

Comment:
With this enabled, AdGuard protection will be paused while Samsung Pay is in use. Some of our users require this feature as they experience issues with Samsung Pay when AdGuard is enabled.